### ISSUE TITLE
Auto-migrate old keystores on startup

### ISSUE THREAD

**Issue by kentbull** [WebOfTrust/keria#244]
On startup then agents should be checked for needing migrations. When an old keystore is detected, as in the `stts.` database is a `dict` rather than a `KeyStateRecord`, then KERIA should automatically update the agent's database.

**Comment by pfeairheller** [comment]
I do not believe this to be wise.  Migrating production systems without intentionally doing so is a mistake.

**Comment by kentbull** [comment]
Agreed, closing the issue. Detection is fine, yet auto-migration is not. That should be explicit.
